### PR TITLE
Handle pages created outside of events

### PR DESCRIPTION
**Pull request by danhenderson**
Rather than pass in defaults in via the create page form, just setting some nicer defaults to handle creating a page in it's own event.

Is `eventId` passed in?
- Yes: forward that through in the request and add the page to the event
- No: pass in `activityType` (with an "othercelebration" default), `eventName` (defaulting to the page name) and `eventDate` (no current default)

### CODE REVIEW

File: source/api/pages/justgiving/index.js [hunk 1]
```diff
@@ -248,9 +256,7 @@ export const createPage = ({
         consistentErrorResponses,
         currency,
         customCodes,
-        eventDate,
         eventId,
```
Comment (rowanhogan) [inline source/api/pages/justgiving/index.js:259]: Should you remove this line?

Comment (danhenderson) [reply source/api/pages/justgiving/index.js:259]: My thinking was yes. The reason being the eventDate is passed through if there is no eventId (now line 246), but if there is no eventId, it is irrelevant (like Kerri's example pages where it just pulls back the actual overarching event's date).

Comment (danhenderson) [reply source/api/pages/justgiving/index.js:259]: Does that sound right?

Comment (rowanhogan) [reply source/api/pages/justgiving/index.js:259]: Yep. Let me rephrase: "Didn't you mean to delete this line?" You left `eventId` as a standalone param.

Comment (rowanhogan) [reply source/api/pages/justgiving/index.js:259]: I mean it shouldn't really matter cause it would be undefined, but it's confusing to leave it there.

Comment (danhenderson) [reply source/api/pages/justgiving/index.js:259]: Oh, I foolishly thought you were pointing to the eventDate line, not the evenId line. You are right, it should be removed.
